cinemy/cinema/templatetags/filter.py

```diff
@@ -17,3 +17,8 @@ def add_str(arg1, arg2):
 def format_time(input):
     input = int(input)
     return str(input // 60) + "h:" + str(input % 60) + "m"
+
+
+@register.filter(name="get_url")
+def get_url(input):
+    return "https://www.youtube.com/embed/" + input.split("=")[-1]
```

Comment:
Not sure I understand how the embed text is processed because, for example I get this
'<iframe width="560" height="315" src="https://www.youtube.com/embed/9XH2ErSdHBs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>'
and applying input.split("=")[-1] to it, I get '"accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>'

At a first look it looks easier to just save the youtube id in the DB and generate the embedded url.

Reply:
My bad, the trailer_url doesn't have the full embed text I guess, but still, this is confusing as I'm not sure what the trailer_url field holds in it.
Maybe it would be useful to add docstrings in the methods, views...

Reply:
The trailer url contains the exact link from the youtube (copy-pasted from the URL bar). With this filter I am generating the embedded link.

Reply:
Both of the below URLs are valid but I guess they do a redirect so the admin should have the latest URL format.
https://www.youtube.com/watch?v=7nTxdl29ePY
https://www.youtube.com/watch/9XH2ErSdHBs
Having just the ID would be lighter (on the DB) and easier to handle but this will do, as well. No need to change anything.
Thanks!